tatarenko245/procuriosity_autotests
# Create EI (Check EI release)

TestCases:
1. Check status code and message from Kafka topic after EI creation;
2. Check EI release data after Ei creation based on full data model;
3. Check EI release after Ei creation on model without optional fields;
4. Check EI release data after Ei creation based on full data model with 3 items objects.
